src/redux/reducers/experiments/experimentsLoaded.js

```diff
@@ -1,18 +1,20 @@
 const experimentsLoaded = (state, action) => {
   const { experiments } = action.payload;
 
-  const newExperiments = experiments.reduce((acc, curr) => ({
-    ...acc,
-    ids: [...acc.ids, curr.experimentId],
-    [curr.experimentId]: {
+  const newExperiments = experiments.reduce((acc, curr) => {
+    acc.ids.push(curr.experimentId);
+
+    acc[curr.experimentId] = {
       projectUuid: curr.projectId,
       name: curr.experimentName,
       description: curr.description,
       id: curr.experimentId,
-      createdAt: curr.createdAt,
+      createdDate: curr.createdDate,
       lastViewed: curr.lastViewed,
-    },
-  }), { ids: [...state.ids] });
+    };
+
+    return acc;
+  }, { ids: [] });
```

Comment:
I am under the impression that we are loosing pre-existing ids 🤔 
That said, I think that I would rewrite this using a `forEach` or `for (const experiment of experiments)...` updating an `ids` and a `newExperiments` variable.

Reply:
Oh right! Thanks!